apps/client-ui/scripts/generate-amplify-config.sh

```
@@ -0,0 +1,371 @@
+    return 0
+}
+
+# Function to substitute environment variables in template
+substitute_variables() {
+    local template_file="$1"
+    local output_file="$2"
+    
+    print_debug "Substituting variables in template"
+    
+    # Create a temporary file for processing
+    local temp_file=$(mktemp)
```

Shellcheck (0.10.0)

[warning] Declare and assign separately to avoid masking return values.

(SC2155)